### DISCUSSION TITLE
Issues downloading data on Cognition.run

### DISCUSSION THREAD

**Discussion by lmarusich** [jspsych/jsPsych#2464] [Q&A] [answered]
After doing "Delete all data" multiple times on early test runs and then collecting data from a new batch of participants, the data previewed on the Cognition.run site looks correct, but when we attempt to download all the data, the resulting file still includes the old deleted data and is missing some of the new data. It seems like deleted Run IDs may be re-used but not overwritten in the downloaded data.

**Comment by jodeleeuw** [comment]
Tagging @javidalpe on this since it sounds like a cognition.run specific issue.

**Comment by javidalpe** [comment, marked as answer]
Thanks @lmarusich for reporting this issue.

The data was successfully deleted, and new data was recorded. The issue was the browser cache. New recorded CSV files were ignored and deleted files were downloaded instead(from browser cache).

I have updated the cache policy of the downloader tool to ignore browser cache.

**Reply by lmarusich** [reply]
Thank you!
